spec/assessments/metaDescriptionKeywordAsssessment.js

```diff
@@ -0,0 +1,28 @@
+var metaDescriptionKeyword = require( "../../js/assessments/metaDescriptionKeyword.js" );
+var Paper = require( "../../js/values/Paper.js" );
+
+var factory = require( "../helpers/factory.js" );
+var i18n = factory.buildJed();
+
+describe( "the metadescription keyword assessment", function() {
+	it( "should score a meta with no matching keyword as bad", function() {
+		var paper = new Paper();
```

Comment:
You can use `beforeEach` to do something before each test. Such as creating an empty paper object.